### DISCUSSION TITLE
How to retrieve recordings and transcripts after receiving a notification?

### DISCUSSION THREAD

**Discussion by MikeYeager** [AzureAD/microsoft-identity-web#2833] [Q&A]
I was able to successfully create notification subscriptions for recording creation and transcript creation. The notification contains resource property that looks something like this (shortened for brevity):

"resource": "communications/onlineMeetings('MSo5NzZmN ... XhAdGhyZWFkLnYy')/recordings('VjIjIzE5NzZmNG ... JlNzI=')"

and the docs say I can use this to download the recording (or transcript). But it seems this is a portion of a URL. Is there some way to use GraphServiceClient to do the download or do I need to make an HTTPS call using an HTTPClient? The only way I can see to download a recording or transcript with GraphServiceClient is if I have the MeetingId and RecordingId (or TranscriptId), which are not included in the notification.

Thanks in advance, Mike

**Comment by jmprieur** [comment]
@MikeYeager  think it's more a question for the Microsoft Graph SDK

**Comment by MikeYeager** [comment]
@jmprieur I will try over there. I'm honestly confused how the MS Graph library ended up in this namespace. I realize it's combining token acquisition with MS Graph functionality, but the old Microsoft.Graph.Auth namespace made more sense. Thanks!
